dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowFactoryImpl.java

```diff
@@ -775,23 +792,27 @@ public boolean existSchemeIdOnSchemesList(String schemeId, List<WorkflowScheme>
 	@Override
 	public WorkflowTask findTaskByContentlet(final Contentlet contentlet) throws DotDataException {
 		if (!UtilMethods.isSet(contentlet.getIdentifier()) || cache.is404(contentlet)) {
-			return new WorkflowTask();
+			return null;
 		}
 
 		WorkflowTask task = cache.getTask(contentlet);
 
 		if (task == null) {
-			final HibernateUtil hu = new HibernateUtil(WorkflowTask.class);
-			hu.setQuery(
-					"from workflow_task in class com.dotmarketing.portlets.workflows.model.WorkflowTask where webasset = ? and language_id = ?");
-			hu.setParam(contentlet.getIdentifier());
-			hu.setParam(contentlet.getLanguageId());
-			task = (WorkflowTask) hu.load();
-
-			if (task != null && task.getId()!=null) {
-				cache.addTask(contentlet, task);
+
+			final DotConnect db = new DotConnect();
+			db.setSQL(WorkflowSQL.SELECT_TASK);
+			db.addParam(contentlet.getIdentifier());
+			db.addParam(contentlet.getLanguageId());
+
+			List<WorkflowTask> foundTasks = this
+					.convertListToObjects(db.loadObjectResults(), WorkflowTask.class);
+			if (null != foundTasks && !foundTasks.isEmpty()) {
+				task = foundTasks.get(0);
 			}
-			else{
+
+			if (null != task && null != task.getId()) {

```

Comment:
format this

Reply:
@jdotcms this code is formatted. What do you want to  change?